scripts/macos-gen-dmg.py

```diff
@@ -26,6 +26,19 @@ def main(args):
         nargs='?',
     )
 
+    parser.add_argument('--universal',
+        help='Universal build',
+        default=False,
+        action=argparse.BooleanOptionalAction,
+    )
+
+    parser.add_argument('combine_dir',
+        help='The build directory (defaults to CWD)',
+        default=Path(os.getcwd()),
+        type=Path,
+        nargs='?',
+    )
+
```

Comment:
I think it makes more sense to have `combine_dir` as an argument to `--universal`, because it's meaningless without that option. Not even sure if it should be optional.